src/App.tsx

```diff
@@ -81,13 +81,13 @@ function App(): ReactElement {
 
     return (
         <>
-            <a className="skip__link" href="#main-content">Skip to main content</a>
+            <a className="ons-skip__link" href="#main-content">Skip to main content</a>
```

Comment:
👋
In the new CSS file, I think they changed skip__link to skip-link
I assume they're the same thing just different names

Reply:
Thanks Kris, I've changed this to ons-skip-link